app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,28 @@
 <resources>
     <string name="app_name">JavaBaseProject</string>
-    <string name="activity_main_button_text">PRESS ME</string>
+    <string name="activity_main_set_button_text">SET SPOTS</string>
+    <string name="activity_main_reservation_button_text">MAKE RESERVATION</string>
     <string name="activity_main_title_text">AVAILABLE PARKING SPOTS</string>
-    <string name="activity_main_available_text">AVAILABLE SPOTS: \n%s</string>
+    <string name="activity_main_available_text">%d</string>
```

Comment:
It does not need to create a resource for this value

Reply:
fixed